README.md

```diff
@@ -43,6 +43,12 @@ Login credentials: `admin@example.com`/`admin`
 * Follow us on [Twitter](https://twitter.com/getsaleor?lang=en)
 * Check our latest blog posts on [Medium](https://medium.com/saleor)
 
+## 🗺 Translate
```

Comment:
It's hard to say which emoji this is, I think that earth emoji 🌎 would be more readable

https://emojipedia.org/earth-globe-europe-africa/

Reply:
America 🌎 or Europe-Africa 🌍?

Reply:
America is better-looking, but that's the matter of preferences :smirk:

Reply:
Less details shown :+1: